python/ray/data/read_api.py

```diff
@@ -2282,6 +2283,9 @@ def from_tf(
 @PublicAPI
 def from_torch(
     dataset: "torch.utils.data.Dataset",
+    parallelism: int = -1,
+    random_split: bool = False,
+    batch_size: int = 32,
```

Comment:
Let's add the new args to the docstring?

Also I don't think we want to expose batch_size and random_split? Should it be shuffle instead of random_split?

Reply:
random_split doesn't actually shuffle the dataset, it just partitions the dataset randomly. The order within those partitions remain the same in the final dataset. @scottjlee recommended changing the name to `shuffle_blocks`

Reply:
We should shuffle rows instead of shuffling blocks.

Create a list of integers from 0 to len(dataset).
Shuffle this list of integers
Then partition this list of integers evenly across all the read tasks

Reply:
Actually, I read wrong, random_split does not maintain relative order inside splits, so this is a true shuffle.